content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -10,7 +10,7 @@ content_template: templates/task
 
 {{% capture overview %}}
 This page shows how to safely drain a machine, respecting the application-level
-disruption SLOs you have specified using PodDisruptionBudget.
+disruption SLOs (Service-Level Objectives) you have specified using PodDisruptionBudget.
```

Comment:
Don't capitalized the exploded acronym. Also, would it be better to remove the acronym and rephrase it to "respecting the PodDisruptionBudget you have defined"?

Reply:
I knew that rule at some point, thank you for the reminder.

I prefer your rephrase suggestion. This wording extends the PodDisruptionBudget definition with this specific drain example. Since this page is linked from the "Disruptions" page in a "Learn more" context, I think it fits perfectly.

Reply:
I like it. Please make the change and ping me for re-review when it's done. Thanks!